### ISSUE TITLE
Human readable cron

### ISSUE THREAD

**Issue by claudiuconstantin** [vincentjames501/angular-cron-gen#16]
Users of `angular-cron-gen` library would most probably benefit a lot from a "human readable" explanation option of a cron result. Example:  `0 0 11 1/1 * ? *` could be easily displayed as `At 01:00 AM, only on Monday and Wednesday`.

Would an integration with libraries like [cronstrue](https://www.npmjs.com/package/cronstrue) be a good idea? I could gladly submit a PR for this.

**Comment by vincentjames501** [comment]
Yeah, I'd love to see it! In particular on the advanced page would be really useful as well.

**Comment by vincentjames501** [comment]
It may be interesting to explore this for cron validation as well. Right now I'm using a massive regex which isn't quartz complete.
